src/tuesday/core/plotting/powerspectra.py

```diff
@@ -88,6 +88,107 @@ def plot_1d_power_spectrum(
     return ax
 
 
+def plot_1d_power_spectrum_z(
+    power_spectrum: list[SphericalPS],
+    *,
+    ax: plt.Axes | None = None,
+    title: str | None = None,
+    xlabel: str | None = None,
+    ylabel: str | None = None,
+    at_k: float | None = None,
+    color: list | None = None,
+    log: list[bool] | None = False,
+    fontsize: float | None = 16,
+    legend: str | None = None,
+    smooth: float | bool = False,
+    legend_kwargs: dict | None = None,
+) -> tuple[plt.Figure, plt.Axes]:
+    """
+    Plot a 1D power spectrum.
+
+    Parameters
+    ----------
+    power_spectrum : list[SphericalPS]
+        List of instances of the SphericalPS class.
+    ax : plt.Axes, optional
+        Axes object to plot on. If None, a new axes is created.
+    title : str, optional
+        Title of the plot.
+    xlabel : str, optional
+        Label for the x-axis.
+    ylabel : str, optional
+        Label for the y-axis.
+    at_k : float | int, optional
+        If provided, plots the 1D power spectrum at a specific k value.
+        The k value is assumed to be in the same unit
+        as the k in the SphericalPS instance wavemodes.
```

Comment:
Just make this required -- I think by default plotting at the lowest k is just weird.